website/server/middlewares/errorHandler.js

```diff
@@ -83,6 +83,15 @@ module.exports = function errorHandler (err, req, res, next) { // eslint-disable
 
   if (responseErr.errors) {
     jsonRes.errors = responseErr.errors;
+    if (jsonRes.message === res.t('invalidReqParams')) {
```

Comment:
So there's a problem with this change, it shouldn't change the actual message returned, the API already returns the specific error messages in the `errors` property of `jsonRes`, you should use its content to show a different message on the client side instead of changing the server side message